### PR TITLE
Fix missing null checks

### PR DESCRIPTION
**Pull request by mattcaswell**
##### Checklist

- [x] CLA is signed

### CODE REVIEW

File: ssl/t1_lib.c [hunk 1]
```diff
@@ -1538,6 +1538,10 @@ static int add_client_key_share_ext(SSL *s, WPACKET *pkt, int *al)
     }
 
     skey = ssl_generate_pkey(ckey);
+    if (skey == NULL) {
+        SSLerr(SSL_F_ADD_CLIENT_KEY_SHARE_EXT, ERR_R_MALLOC_FAILURE);
```
Comment (richsalz) [inline ssl/t1_lib.c]: why does this set the error code but the ones in statem_clnt.c do not?

Comment (mattcaswell) [inline ssl/t1_lib.c]: The first one in statem_clnt.c does (its in the err block). I'll add one for the other one.

**Comment by mattcaswell** [comment]
Updated commits pushed.

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**